testing/scenario_app/README.md

```diff
@@ -56,11 +56,20 @@ compared against golden reside.
 
 ## Running for Android
 
-The only test that is currently run is the Firebase TestLab test, which is a
-smoke test to make sure an application can be built and run on a real Android
-device.
+### Integration tests
 
-To run it, build the `android_profile_arm64` variant of the engine, and run
+For emulators running on a x64 host, build `android_debug_unopt_x64` using
+`./tools/gn --android --unoptimized --goma --android-cpu=x64`.
+
+Then, launch the emulator, and run `./testing/scenario_app/run_android_tests.sh android_debug_unopt_x64`.
```

Comment:
Please don't introduce a shell script for this - use python.

Reply:
this is the existing shell script. It just calls dart with some flags. Is there a concern with this shell script?

Reply:
I guess I'd just make sure it's actually used on CI. I think it's not, but if I'm wrong then leave it. If it's not then we should drop it.